src/test/java/com/driving/school/service/stub/StubCrudService.java

```diff
@@ -0,0 +1,17 @@
+package com.driving.school.service.stub;
+
+import com.driving.school.dto.mapper.GenericMapper;
+import com.driving.school.service.CrudService;
+import com.driving.school.service.util.RemovalUtil;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public class StubCrudService extends CrudService<String> {
```

Comment:
Rozwiń proszę nazwę tej klasy

Reply:
https://en.wikipedia.org/wiki/Test_stub
to jest potrzebne do testowania tej abstrakcyjnej CrudService (bo zwykle nie możesz testować klas abstrakcyjnych, ale u nas jest tam dużo metod domyślnych, więc się opłaca).